src/pages/SubmissionConfig.js

```diff
@@ -156,14 +160,16 @@ export const SubmissionConfig = ({ methodId }) => {
       if (!wdsProxyUrl || (wdsProxyUrl.status !== 'Ready')) {
         const { status, state: wdsUrlRoot } = await loadWdsUrl()
         if (status === 'Unauthorized') {
-          notify('warn', 'Error loading data tables', { detail: 'Service returned Unauthorized error. Session might have expired. Please close the tab and re-open it.' })
+          notify('warn', 'Error loading data tables',
+            { detail: 'Service returned Unauthorized error. Session might have expired. Please close the tab and re-open it.' })
```

Comment:
Weird spacing? Was this intentional?

Reply:
I think this happened on save, which is strange because it should be using the same settings as lint (and this passed the linter)